src/data/nat/basic.lean

```diff
@@ -1367,6 +1367,10 @@ by { convert bit1_lt_bit0_iff, refl, }
 | ff := bit0_le_bit1_iff
 | tt := bit1_le_bit1
 
+@[simp] lemma bit0_mod_two : bit0 n % 2 = 0 := by rw nat.mod_two_of_bodd; simp
```

Comment:
```suggestion
@[simp] lemma bit0_mod_two : bit0 n % 2 = 0 := by { rw nat.mod_two_of_bodd, simp }
```

Reply:
Is there a reason to prefer the braces?  I thought semicolons are ok now that Lean displays the goal state after them?

Reply:
If semicolons are OK, nobody has told that to the official style guide yet :) I slightly prefer commas because they suggest that `rw` doesn't create extra goals. Not really a make-or-break problem otherwise.

Reply:
Ok, I've made a PR: https://github.com/leanprover-community/leanprover-community.github.io/pull/135